projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -52,6 +52,10 @@
 			flex: 0 0 100%;
 			margin: 0;
 
+			&.wp-block-jetpack-button {
+				flex-basis: auto;
```

Comment:
Overwrite the `100%` value. We want to be able to set the width through the button styling options in the editor.